pkgs/by-name/tm/tmc-cli/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
This way you can reuse attrs